warehouse/models/intermediate/gtfs/gtfs_quality/int_gtfs_quality__schedule_feed_guideline_index_aggregator.sql

```diff
@@ -0,0 +1,20 @@
+{{ config(materialized='ephemeral') }}
+
+WITH int_gtfs_quality__schedule_feed_guideline_index AS (
+    SELECT * FROM {{ ref('int_gtfs_quality__schedule_feed_guideline_index') }}
+    -- 1/13/23 is when the aggregator scraper started running
+    WHERE date >= '2023-01-13'
+),
+
+distinct_aggregators AS (
+    SELECT DISTINCT aggregator
+      FROM {{ ref('stg_gtfs_quality__scraped_urls') }}
+),
```

Comment:
nit: I think you could pull the minimum date from stg_gtfs_quality__scraped_urls and use that to filter the index